spec/controllers/user_invite_requests_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "spec_helper"
+
+describe UserInviteRequestsController do
+  include RedirectExpectationHelper
+  include LoginMacros
+
+  authorized_roles = %w[superadmin policy_and_abuse].freeze
+
+  describe "GET #index" do
+    subject { get :index }
+    let(:success) do
+      expect(response).to render_template("index")
+    end
+
+    it_behaves_like "an action only authorized admins can access", authorized_roles: authorized_roles
+
+    it "does not allow non-admins to view user invite requests" do
```

Comment:
As long as I'm nitpicking, can this say `access` instead of `view`? And should we have a test for logged out users?